exercise5_code.R

```diff
@@ -0,0 +1,23 @@
+setwd("C:/Users/DAVIS/Desktop/shell-novice-data/exer5/")
+data=read.table(file="test.dat",header=FALSE, sep=" ")
+
+wages = read.csv(file="wages.csv", header=TRUE, stringsAsFactors = FALSE)
+
+#Challenge question 1
+test1 <- unique(wages[,c("gender","yearsExperience")])
+test2 <- test1[order(test1[,1], test1[,2]),]
+write.csv(test2,"gender_years_in_order.csv")
+
+#Challenge question 2
+
+
+#Challenge question 3

```

Comment:
The key word in this question is _effect_. So, the correct answer would include the difference between the minimum difference between graduating high school and graduating college.